basic_autonomy/include/basic_autonomy/basic_autonomy.h

```diff
@@ -271,13 +271,12 @@ namespace basic_autonomy
       * \param starting_downtrack The downtrack distance from which the lane change maneuver starts
       * \param ending_downtrack The downtrack distance at which the lane change maneuver end
       * \param wm Pointer to intialized world model for semantic map access
-      * \param state The vehicle state at the time the function is called
-      * \param buffer_ending_downtrack The additional downtrack beyond requested end dist used to fit points along spline
+      * \param downsample_ratio TODO: add description
       * 
       * \return A vector of geometry points as lanelet::basicpoint2d
       */
      std::vector<lanelet::BasicPoint2d> create_lanechange_geometry(lanelet::Id starting_lane_id, lanelet::Id ending_lane_id, double starting_downtrack, double ending_downtrack,
-                                                            const carma_wm::WorldModelConstPtr &wm,const cav_msgs::VehicleState &state, int downsample_ratio, double buffer_ending_downtrack);
```

Comment:
I think this merge is suspect, both on develop and master it looks like state is still an input to this function.